src/common/packets/packets2020_len_main.h

```diff
@@ -2,8 +2,8 @@
  * This file is part of Hercules.
  * http://herc.ws - http://github.com/HerculesWS/Hercules
  *
- * Copyright (C) 2018-2020 Hercules Dev Team
- * Copyright (C) 2018-2020 Andrei Karas (4144)
+ * Copyright (C) 2018-2020  Hercules Dev Team
+ * Copyright (C) 2018-2020  Andrei Karas (4144)
```

Comment:
unnecessary empty space

Reply:
i think here should be space

Reply:
it become double space

Reply:
I recently removed the extra space from the other files, but I don't have a strong opinion on whether there should be one or two (Looking around in copyright headers from other projects, it's usually one though)

Reply:
i updated template in generation scripts to one space.
this mean update for next client will use one space here